src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,79 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.People;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(w -> w.split(","))
+                .flatMap(Arrays::stream)
+                .mapToInt(Integer::parseInt)
+                .filter(value -> value % 2 == 0)
+                .min()
+                .orElseThrow(
+                        () -> new RuntimeException("Can't get min value from list:"
+                                + numbers.toString()));
```

Comment:
```suggestion
                .orElseThrow((() -> new RuntimeException("Can't get min value from list " + numbers)));
```
toString is redundant